src/codeu/chat/common/BasicController.java

```diff
@@ -54,4 +54,17 @@
   //  Whether conversations can have the same title is undefined.
   Conversation newConversation(String title, Uuid owner);
 
+  // RENAME CONVERSATION
+  //
+  // Rename an existing conversation on the server. All parameters must
+  // be provided or else the server won't apply the change. If the operation
+  // is successful, the existing conversation's name will be renamed to
+  // the user-provided name.
+  Conversation renameConversation(Uuid id, String newTitle);
+
+  // DELETE CONVERSATION
+  // Deletes an existing conversation on the server. All parameters must
+  // be provided or else the server won't apply the change. If the operation
+  // is successul, the existing conversation will be removed from the server
+  void removeConversation(Uuid id);
```

Comment:
Usually we want to return the conversation that was deleted, or at least a boolean for success